lib/error.js

```diff
@@ -41,8 +41,21 @@ class HttpStreamError extends Error {
     }
 }
 
+class IllegalArgumentError extends Error {
+    constructor(message, value) {
+        if (value === undefined) {
```

Comment:
Is `value = null` an authorized value?

Reply:
That will show up as `message: null (object)` below, however for `undefined` it shows up as `message: undefined (undefined)` which I thought was a little silly.